packages/core/src/components/icon/icon.md

```diff
@@ -67,7 +67,7 @@ import { Icon, IconSize } from "@blueprintjs/core";
     <h4 class="@ns-heading">Some icons are missing in the icon font in 3.x</h4>
 
 Blueprint icons added after `@blueprintjs/icons@3.14.0` do not have icon font support.
-Support for the icon font for these icons will be restored in Blueprint v4.0.
+Support for the icon font for these icons will be restored in Blueprint v5.0.
```

Comment:
please revert this line; we're dealing with the icon font (either restoring support or removing it entirely) in v4.0